js_modules/dagster-ui/packages/ui-core/src/assets/auto-materialization/AutomaterializationRoot.tsx

```diff
@@ -124,7 +124,7 @@ const GlobalAutomaterializationRoot = () => {
       return (
         ticks?.map((tick, index) => {
           // For ticks that get stuck in "Started" state without an endTimestamp.
-          if (index !== 0 && !isOldTickWithoutEndtimestamp(tick) && !tick.endTimestamp) {
+          if (index !== 0 && !isOldTickWithoutEndtimestamp(tick, index) && !tick.endTimestamp) {
```

Comment:
I think that the index !== 0 && and  !tick.endTimestamp parts of this if satement are duplicated inside the isOldTickWithoutEndtimestamp fn now, maybe we should de-dupe?

Reply:
It actually wasn't duplicated 😓